evm_loader/program/src/neon.rs

```diff
@@ -0,0 +1,38 @@
+//! NEON ELF
+#![allow(clippy::use_self,clippy::nursery)]
+
+use const_format::formatcp;
+use crate::account_data::ACCOUNT_SEED_VERSION;
+use crate::account_data::ACCOUNT_MAX_SIZE;
+
+
+macro_rules! str_as_bytes_len {
+    ($value:expr) => {
+        {
+            $value.as_bytes().len()
+        }
+    }
+}
+
+macro_rules! neon_elf {
```

Comment:
The name is not clear. `neon_elf_param` will be better. And you need to add a description of what this macro is used for.